lib/gauge.js

```diff
@@ -1,15 +1,19 @@
 var os = require('os');
 
-module.exports = function(influxclient) {
+module.exports = function(metricsClient, type) {
```

Comment:
This type is mainly for statsd because it have a few different types for things like counter, timer, even though they are gauges, statsd uses different type